File: templates/tests-php/BREACH-005.php
<?php

namespace Compliance\Tests;

use PHPUnit\Framework\TestCase;

class MFATest extends TestCase
{
    public function testVerifyMFAEnabled(): void
    {
        $config = Config::load();
        $this->assertEquals('Enabled', $config->authentication->mfa);
    }
}

File: templates/tests-csharp/BREACH-005.cs
using Xunit;

namespace Compliance.Tests
{
    public class MFATests
    {
        [Fact]
        public void TestVerifyMFAEnabled()
        {
            var config = Config.Load();
            Assert.Equal("Enabled", config.Authentication.Mfa);
        }
    }
}
